build.gradle

```diff
@@ -92,37 +90,36 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
     implementation("org.springframework.boot:spring-boot-starter-cache")
 
-    implementation 'org.springframework.data:spring-data-redis:2.2.3.RELEASE'
-    implementation 'redis.clients:jedis:3.1.0'
+    implementation 'org.springframework.data:spring-data-redis:2.2.4.RELEASE'
+    implementation 'redis.clients:jedis:3.2.0'
 
-    implementation("org.springframework.cloud:spring-cloud-starter-aws-messaging:2.2.0.RELEASE")
-    implementation 'org.springframework:spring-jms'
+    implementation("org.springframework.cloud:spring-cloud-starter-aws-messaging:2.2.1.RELEASE")
+    implementation 'org.springframework:spring-jms:5.2.3.RELEASE'
     implementation 'com.amazonaws:amazon-sqs-java-messaging-lib:1.0.8'
     implementation 'org.apache.camel:camel-spring-boot:2.23.2'
 
     implementation("org.springframework.boot:spring-boot-starter-security")
     implementation("org.springframework.security.oauth:spring-security-oauth2:2.4.0.RELEASE")
     implementation("org.springframework.security:spring-security-jwt:1.1.0.RELEASE")
 
-    implementation 'org.apache.camel:camel-spring-boot:2.24.2'
-    implementation 'org.apache.camel:camel-csv:2.24.2'
-    implementation ('org.apache.camel:camel-aws:2.24.2') {
+    implementation 'org.apache.camel:camel-spring-boot:2.24.3'
+    implementation 'org.apache.camel:camel-csv:2.24.3'
+    implementation ('org.apache.camel:camel-aws:2.24.3') {
         exclude group: 'com.amazonaws', module: 'aws-java-sdk-sagemaker'
         exclude group: 'com.amazonaws', module: 'aws-java-sdk-sagemakerruntime'
         exclude group: 'com.amazonaws', module: 'aws-java-sdk-opsworkscm'
-
     }
-    implementation 'net.javacrumbs.shedlock:shedlock-spring:3.0.0'
-    implementation 'net.javacrumbs.shedlock:shedlock-provider-jdbc-template:3.0.0'
+    implementation 'net.javacrumbs.shedlock:shedlock-spring:4.2.0'
+    implementation 'net.javacrumbs.shedlock:shedlock-provider-jdbc-template:4.2.0'
 
-    implementation 'net.logstash.logback:logstash-logback-encoder:6.2'
+    implementation 'net.logstash.logback:logstash-logback-encoder:6.3'
     implementation 'com.microsoft.azure:applicationinsights-spring-boot-starter:2.6.0-BETA'
     implementation 'com.microsoft.azure:applicationinsights-logging-logback:2.6.0-BETA'
     agentDeps 'com.microsoft.azure:applicationinsights-agent:2.6.0-BETA'
 
     implementation 'javax.annotation:javax.annotation-api:1.3.2'
-    implementation 'javax.xml.bind:jaxb-api:2.3.0'
-    implementation 'com.sun.xml.bind:jaxb-impl:2.3.2'
+    implementation 'javax.xml.bind:jaxb-api:2.4.0-b180830.0359'
```

Comment:
Will these -b prefixes mean they are beta versions?

Reply:
Ive just gone with :dependencyUpdates's recommendations

Reply:
agreed, doesn't seem right to me.  Looks like this project doesn't have the latest dependency update configuration so is suggesting beta versions too.  Worth therefore copying the configuration from a different project - e.g. nomis-oauth2-server.  Even so it sometimes suggests beta versions which we then tend to ignore ..

Reply:
Done.